log_analytics.tf

```diff
@@ -0,0 +1,65 @@
+resource "azurerm_log_analytics_workspace" "main" {

```

Comment:
How this change about `azurerm_log_analytics_workspace` is related with the role assignments for ingress ? Should this go to a different PR ?